src/entity-manager/EntityManager.ts

```diff
@@ -496,7 +495,14 @@ export class EntityManager {
         const metadata = this.connection.getMetadata(entityClass);
         const qb = this.createQueryBuilder(entityClass, FindOptionsUtils.extractFindManyOptionsAlias(optionsOrConditions) || metadata.name);
         FindOptionsUtils.applyFindManyOptionsOrConditionsToQueryBuilder(qb, optionsOrConditions);
-        this.joinEagerRelations(qb, qb.alias, metadata);
+
+        ids = ids.map(id => {
```

Comment:
why do you need those changes? Looks like an old code from the `master` branch. Are you sure you did a proper branch from next and not from master?

Reply:
Ah, rebase gone wrong. I branched from master at first, then rebased with next. Fixed